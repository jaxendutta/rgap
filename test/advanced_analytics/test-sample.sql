SELECT 
  YEAR(rg.agreement_start_date) AS year,
  o.org AS funding_agency,
  COUNT(*) AS total_grants,
  SUM(rg.agreement_value) AS total_funding,
  AVG(rg.agreement_value) AS avg_funding
FROM ResearchGrant rg
JOIN Organization o ON rg.org = o.org
GROUP BY YEAR(rg.agreement_start_date), o.org
ORDER BY year, total_funding DESC;
