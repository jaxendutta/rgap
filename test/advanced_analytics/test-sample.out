+------+----------------+--------------+---------------+---------------+
| year | funding_agency | total_grants | total_funding | avg_funding   |
+------+----------------+--------------+---------------+---------------+
| 2016 | CIHR           |         1671 | 1173216208.00 | 702104.253740 |
| 2016 | SSHRC          |            1 |     112500.00 | 112500.000000 |
| 2017 | CIHR           |         1732 | 1016957222.00 | 587157.749423 |
| 2017 | SSHRC          |         3329 |  794419407.65 | 238636.049159 |
| 2017 | NSERC          |        10652 |  791677701.14 |  74321.977201 |
| 2018 | SSHRC          |         5982 | 1223739196.43 | 204570.243469 |
| 2018 | NSERC          |        13284 | 1065149917.18 |  80182.920595 |
| 2018 | CIHR           |         3189 | 1063322501.00 | 333434.462527 |
+------+----------------+--------------+---------------+---------------+
8 rows in set (0.25 sec)
